### PR TITLE
Added Sunspot Data Importing

### PR DESCRIPTION
**Pull request by abotiamnot**
Added an additional functionality to import sunspot data from the following data source, 

http://www.sidc.be/silso/datafiles

To import the data, 
`import heliopy.data.sunspot as sunspot`
and then call the data function, 
`sunspot.data()`

### CODE REVIEW

File: heliopy/data/sunspot.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import requests  # To download file
+import datetime  # To find current date/time
+import pandas  # For Pandas Dataframe
+import os  # For extra stuff
```
Comment (dstansby) [inline heliopy/data/sunspot.py]: These imports don't need the comments.

File: heliopy/data/sunspot.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import requests  # To download file
+import datetime  # To find current date/time
+import pandas  # For Pandas Dataframe
+import os  # For extra stuff
+
+data_source = 'http://www.sidc.be/silso/INFO/sndtotcsv.php'  # File Link
+header_ = ['Y', 'M', 'D', 'DecD', 'Daily', 'Std Dev', 'No Obs', 'Def/Prov Ind']
+
+
+def data():
```
Comment (dstansby) [inline heliopy/data/sunspot.py]: Can you rename this to `daily()`. It also needs a docstring directly underneath, something like "Daily sunspot number import. See http://www.sidc.be/silso/infosndtot for more details"

File: heliopy/data/sunspot.py (hunk 1 above)
Comment (abotiamnot) [reply heliopy/data/sunspot.py]: Got it, removing them.

File: heliopy/data/sunspot.py (hunk 2 above)
Comment (abotiamnot) [reply heliopy/data/sunspot.py]: Oh yes, I'll add more functions to import different datasets as well, like monthly() and yearly()

**Pull request closed without merging**